.github/workflows/ci.yaml

```diff
@@ -66,6 +74,16 @@ jobs:
             pydantic=${{ matrix.pydantic-version }}
           init-shell: bash
 
+      - name: "Install OpenEye"
```

Comment:
Adding back oechem but only for the one runner (because it's very time consuming)